rx-android-extensions/src/main/java/com/appunite/rx/android/util/LogTransformer.java

```diff
@@ -0,0 +1,75 @@
+package com.appunite.rx.android.util;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import javax.annotation.Nonnull;
+
+import rx.Observable;
+import rx.functions.Action0;
+import rx.functions.Action1;
+
+public class LogTransformer {
+
+    public interface Logger {
+
+        Logger DEFAULT = new Logger() {
+            @Override
+            public void logNext(@NonNull String tag, @Nonnull String message) {
+                Log.i(tag, message);
```

Comment:
i would rather use:
```java
Logger logger = Logger.getLogger(tag);
if (logger.isLogable(Level.VERBOSE)) {
  logger.log(Level.VERBOSE, obseravbleTag + " -> onNext: " + value); 
}
```

so you will never get instance creation and expensive toString when logging is off